### PR TITLE
Make negative queued bookings

### PR DESCRIPTION
**Pull request by raducastravet**
https://etvas.atlassian.net/browse/ETV-9969

### CODE REVIEW

File: src/routes/backoffice.ts [hunk 1]
```diff
@@ -382,11 +382,11 @@ export const processQueuedBooking = async (
...
       };
     }
-
-    // direct debits come with a negative value
-    booking.amount.value = -Math.abs(booking.amount.value);
   }
 
+  // all queued bookings should have negative amounts
+  booking.amount.value = -Math.abs(booking.amount.value);
+
```
Comment (raducastravet) [inline src/routes/backoffice.ts]: This was approved by Decebal

Comment (ohnu) [reply src/routes/backoffice.ts]: yeah .. but do it as a rule in the UI .. not in the BE code

Comment (raducastravet) [reply src/routes/backoffice.ts]: I don't think the effort of doing this on the FE is worth it.

Comment (hututudor) [reply src/routes/backoffice.ts]: https://github.com/etvascom/mock-solaris/pull/24/commits/40534eb2e59250ecec43dd60d5e9ba9e93203ecd

**Review by ohnu — APPROVED** [review]

**Pull request merged**